src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -354,6 +354,10 @@ private static bool IsFloatingPointExpressionOfUnknownPrecision(BoundExpression
                         _factory.Syntax = oldSyntax;
                         return new BoundDelegateCreationExpression(syntax, argument: receiver, methodOpt: method, isExtensionMethod: oldNode.IsExtensionMethod, type: rewrittenType);
                     }
+
+                case ConversionKind.ImplicitNew:
+                    return rewrittenOperand;
+
```

Comment:
Is this branch exercised by tests? I'm wondering if it is reachable.

Reply:
Now that we directly return the bound object creation, it's not reachable. will revert.